testsuite/features/step_definitions/common_steps.rb

```diff
@@ -176,7 +176,7 @@
 # spacewalk errors steps
 Then(/^the up2date logs on client should contain no Traceback error$/) do
   cmd = 'if grep "Traceback" /var/log/up2date ; then exit 1; else exit 0; fi'
-  _out, code = $client.run(cmd)
+  _out, code = $minion.run(cmd)
   raise 'error found, check the client up2date logs' if code.nonzero?
 end
```

Comment:
This was for the client? Do we need if for the minion, too? If yes we should update the name of the step.

Reply:
In theory, SUMA clients are all the clients we register in SUMA.
We can use the word client, the word system, and from now on, we can also use the word minion too :)

But what I will do is clarify this by passing by parameter the host we want to use.